File: Programing_language/java/E1.java
import java.util.Scanner; // (4) Scannerを使うためのimportを追加

public class E1 { 
    public static void main(String[] args) {
        Scanner in = new Scanner(System.in); 
        System.out.println("繰り返し回数を入力して下さい。:");
        int n = in.nextInt(); 
        String word = in.next(); 

        System.out.println("");//改行のための出力 

        for (int i = 0; i < n ; i++) { 
            System.out.println(word); 
        }        
        // 良い慣習として、使い終わったScannerは閉じる
        in.close(); 
    }
}

File: Programing_language/c/E1.c
#include <stdio.h>

int main() {
    int N;
    char word[256];

    printf("繰り返し回数を入力して下さい。\n");
    scanf("%d", &N);

    printf("繰り返す単語を入力して下さい。\n");
    scanf("%255s", word);

    for (int i = 0; i < N; i++) {
        printf("%s\n", word);
    }

    return 0;
}